README.md

```diff
@@ -1,2 +1,36 @@
 # hw01
 Repository for homework 1
+
+Text added through RStudio
+
+A brief biography as a part of the homework:
+
+## Zahra Amalia Syarifah
+I was born in Indonesia and grew up there. However, shortly before my high school graduation I moved to the UK. *So I **never** actually completed high school*. I went straight to a college in Southeast England and moved to Scotland for my undergraduate. Now, I am completing my Master in the States.
+
+For the sake of using the list. Here are the list of my favourite carbohydrates:
+* Pasta
+* Potato
+* Bread
+
+Here is an image of Hungarian Goulash (for the sake of the points, but also because it's one of my favourite dishes):
+
+![Image of Hungarian Goulash](http://img.taste.com.au/JCSA2ji6/w720-h480-cfill-q80/taste/2016/11/goulash-soup-75934-1.jpeg)
+
+You might want to make it at home, so here is a recipe for it:
```

Comment:
Included a link to an outside website. However, it looks as if the link is written as a line separate from a hyperlink. The hyperlink command is correct, so it's needless to have both in the script. This seems like a small mistake missed in the editing process.